docker/docker-compose.lwd.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.8"
+
+services:
+  zebra:
+    ports:
+      # Zebra uses the following inbound and outbound TCP ports
+      - "8233:8233"
+      - "8232:8232"
+    healthcheck:
+      start_period: 1m
+      interval: 15s
+      timeout: 10s
+      retries: 3
+      test: ["CMD-SHELL", "curl --data-binary '{\"jsonrpc\": \"1.0\", \"id\":\"curltest\", \"method\": \"getinfo\", \"params\": [] }' -H 'content-type: application/json' http://127.0.0.1:8232/ || exit 1"]
```

Comment:
Shouldn't this health-check be on *zebra* rather than the lightwalletd service?
Note, this file _says_ it creates a service called `zebra`, but in fact it defines the lightwalletd service) as shown below. So is this healthcheck simply on the wrong file?

Reply:
Before these lines you'll see this:
```yaml
services:
  zebra:
```

This indicates that, when running this YAML, it will override the Zebra service with these values, so the healthcheck is actually added to the Zebra service

Reply:
Interesting. Why not just add them to the zebra service directly?